tests/testthat/test-get_summary.R

```diff
@@ -4,4 +4,200 @@
 #' T1.3 An error when `survfit_object` is a tibble
 #' T1.4 An error when `survfit_object` is a data.table
 #' T1.5 An error when `survfit_object` is a random object
-#' T1.6 An error when `survfit_object` is NULL
\ No newline at end of file
+#' T1.6 An error when `survfit_object` is NULL
+#' T2. The function accepts a survival object
+#' T2.1 An error when `statlist` is NULL
+#' T2.2. An error when `statlist` is a random string
+#' T2.3. The `strata` in the statlist of `get_summary` is the same as `strata` in the `survfit` object
+#' T2.4. The `records` in the statlist of `get_summary` is the same as `n` in the `survfit` object
+#' T2.5. The `events` in the statlist of `get_summary` is the same as events in the `survfit` object
+#' T2.6. The `median` in the statlist of `get_summary` is the same as the median in the `survfit` object
+#' T2.7. The `LCL` in the statlist of `get_summary` is the same as the lower confidence interval in the `survfit` object
+#' T2.8. The `UCL` in the statlist of `get_summary` is the same as the upper confidence interval in the `survfit` object
+#' T2.9. The `CI` in the statlist of `get_summary` is the same as the confidence interval in the `survfit` object
+
```

Comment:
An error when the confidence intervals are requested and not calculated in the survival object

The column for the CI also displays the % of confidence. This is something I added for Alexandra, so it will not display 95% by default. We need a test for this as well.